packages/block-library/src/image/edit.native.js

```diff
@@ -487,6 +547,7 @@ export class ImageEdit extends Component {
 				<PanelBody title={ __( 'Link Settings' ) }>
 					{ this.getLinkSettings( true ) }
 				</PanelBody>
+				{ this.getSetFeaturedButton( isFeaturedImage ) }
```

Comment:
👋 @SiobhyB , what do you think about putting the `androidOnly` conditional here ([example](https://github.com/WordPress/gutenberg/blob/5dec957c24f9afcb5b3e318e179a4cfffa309692/packages/block-library/src/social-links/edit.js#L219)) to avoid the whole running of `getSetFeaturedButton()` on iOS? That way, getSetFeaturedButton get's sligthly cleaner and we also don't run all of its code (like getting styles) unless it will indeed be used (on Android)?

Reply:
Ah, good point! That makes complete sense and I've gone ahead to commit that change.